cmd/main.go

```diff
@@ -15,28 +15,41 @@
 package main
```

Comment:
I think the location of this file is wrong. I think it should remain as is in `/cmd/edgex-ui-go/main.go` and then there should be the `res` folder with `configuration.toml` in it at `/cmd/edgex-ui-go/res/configuration.toml`. 

This is a standard layout for go applications not just for EdgeX but lots of other projects as well, and also scales well when you need to add additional commands in the future.

Reply:
actually after thinking about it, I think the directory name `edgex-ui-server` is a bit redundant (specifically the `go` suffix), I think a better binary name would be `edgex-ui-server`. So then the directory name for this main.go file would be 

```
/cmd/edgex-ui-server/main.go
```

and the compiled executable would be named `edgex-ui-server`. I like this because it's more succinct in explaining what the command does: it's a webserver that presents a UI into edgex

Reply:
ok, it looks properly, i will update it